MaterialLib/SolidModels/KelvinVector-impl.h

```diff
@@ -16,7 +16,7 @@ double Invariants<KelvinVectorSize>::equivalentStress(
     Eigen::Matrix<double, KelvinVectorSize, 1> const& deviatoric_v)
 {
     assert(std::abs(trace(deviatoric_v)) <=
-           1e-16 * std::abs(deviatoric_v.squaredNorm()));
```

Comment:
`deviatoric_v` is a column vector, so `deviatoric_v.diagonal()` is only the first entry of that vector. Is that intended?
Does this small check lead to failing tests/simulations and therefore has to be changed?

Reply:
Seems like I made a mistake ;)